Packs/Phishing/Scripts/FindDuplicateEmailIncidents/FindDuplicateEmailIncidents.py

```diff
@@ -79,10 +79,16 @@ def get_existing_incidents(input_args, current_incident_type):
         get_incidents_args['populateFields'] = ','.join([','.join(fields), input_args['populateFields']])
     else:
         get_incidents_args['populateFields'] = ','.join(fields)
+
+    demisto.debug(f'Calling GetIncidentsByQuery with {get_incidents_args=}')
     incidents_query_res = demisto.executeCommand('GetIncidentsByQuery', get_incidents_args)
     if is_error(incidents_query_res):
         return_error(get_error(incidents_query_res))
-    incidents = json.loads(incidents_query_res[-1]['Contents'])
+    incidents_query_contents = {}
+    for res in incidents_query_res:
+        if res['Contents']:
+            incidents_query_contents = res['Contents']
+    incidents = json.loads(incidents_query_contents)
```

Comment:
since GetIncidentsByQuery always returns at most one entry, why not:
```suggestion
    incidents_query_contents = incidents_query_res[0]['Contents'] if incidents_query_res else '{}'
    incidents = json.loads(incidents_query_contents)
```
btw, note it should be a string `'{}'` and not a dict